src/ffi/c.rs

```diff
@@ -410,13 +440,14 @@ mod c_backend {
     pub use libz::Z_STREAM_END as MZ_STREAM_END;
     pub use libz::Z_STREAM_ERROR as MZ_STREAM_ERROR;
     pub use libz::Z_SYNC_FLUSH as MZ_SYNC_FLUSH;
-    pub type AllocSize = libz::uInt;
 
     pub const MZ_DEFAULT_WINDOW_BITS: c_int = 15;
 
     #[cfg(feature = "zlib-ng")]
     const ZLIB_VERSION: &'static str = "2.1.0.devel\0";
-    #[cfg(not(feature = "zlib-ng"))]
+    #[cfg(feature = "zlib-rs")]
+    const ZLIB_VERSION: &'static str = "0.1.0\0";
```

Comment:
zlib-rs has a `const fn zlibVersion() -> *const c_char`. So maybe these constants could all have type `*const c_char`? the only downside I see is that that is harder to print for debugging, but because these are internal constants that seems fine.

Reply:
It would certainly be nice to automatically obtain the correct value for this, as it would be my guess that these constants quickly go stale otherwise. I also don't know how important these values are, assuming they might be printed under certain circumstances.

Until that's clear, it's probably best to integrate with what's there as is done here.

Reply:
it's important that the major version is the same, (that is what is checked), so the value won't go stale that quickly. Also tests will fail hard when that happens, so there is no real risk of the value going state on the main branch here.